src/entities/judge.ts

```diff
@@ -20,23 +20,23 @@ export class Judge extends BaseEntity {
   @Column({ nullable: true })
   currentTeam?: number;
 
+  @Column({ nullable: true })
+  previousTeam?: number;
+
   async getNextTeam(): Promise<Team> {
     const newTeam = await Team.getNextAvailableTeamExcludingTeams(this.visitedTeams);
     this.currentTeam = newTeam ? newTeam.id : null;
     await this.save();
     return newTeam;
   }
 
-  getLastJudgedTeamId(): number {
-    return this.visitedTeams[this.visitedTeams.length - 1];
-  }
-
   async continue(): Promise<void> {
     await this.recordCurrentTeamAndSave();
   }
 
   async skip(): Promise<void> {
-    await this.clearCurrentTeamAndSave();
+    const updatePrevious = false;
+    await this.recordCurrentTeamAndSave(updatePrevious);
```

Comment:
Is this just to make it clear why it's `false`?

Reply:
Yea that’s exactly why. I figured without it, it just looks like an arbitrary value being passed in and I didn’t want to make the function name super long either.